contracts/StakeManager.sol

```diff
@@ -54,16 +54,19 @@ contract StakeManager is IStakeManager {
         emit StakeWithdrawn(relayManager, msg.sender, amount);
     }
 
-    function authorizeHub(address relayManager, address relayHub) external override {
-        StakeInfo storage info = stakes[relayManager];
-        require(info.owner == msg.sender, "not owner");
+    modifier ownerOrManagerOnly(address relayManager) {
+        StakeInfo storage infoByManager = stakes[relayManager];
+        StakeInfo storage infoBySender = stakes[msg.sender];
+        require(infoByManager.owner == msg.sender || infoBySender.owner != address(0), "not owner or manager");
+        _;
+    }
+
+    function authorizeHub(address relayManager, address relayHub) external override ownerOrManagerOnly(relayManager) {
```

Comment:
This has a security implication that manager account of any hub can now authorise any other hub (therefore can withdraw the stake through the penalisation).
Meaning, there is no longer semantical difference between 'owner' and 'manager'.
Why would manager need a way to authorize a new hub?

Reply:
If we don't have it, no way to auto upgrade server to support new hub. Each upgrade now requires owner manual tx authorization